### PR TITLE
S3 buckets - encryption in transit

### PR DESCRIPTION
**Pull request by QuentinBisson**
Towards https://github.com/giantswarm/giantswarm/issues/29092

### CODE REVIEW

**Comment by QuentinBisson** [comment]
Tested on Gaia 
image

**Review by hervenicol — APPROVED** [review]
Looks good :+1:

File: modules/aws/s3/s3.tf [hunk 1]
```diff
@@ -156,8 +256,8 @@ resource "aws_s3_bucket" "tempo" {
   )
 }
 
-resource "aws_s3_bucket_policy" "access-logs-policy" {
```
[the remark this answers is not in the document]
Comment (QuentinBisson) [reply modules/aws/s3/s3.tf:159]: Oh it was too far from thé bucket though

**Review by whites11 — APPROVED** [review]
ok, but it's unlikely this will be deployed this year

**Comment by QuentinBisson** [comment]
As long as the customer is aware that works for me :)

**Pull request merged**